### PR TITLE
Fixed issue #2924

### PR DESCRIPTION
**Pull request by yunluyl**
Fix bug preventing map interactions near two controls in the same corner #2924

### CODE REVIEW

**Comment by lucaswoj** [comment]
Sorry about the failure! That's an upstream problem that we're fixing in https://github.com/mapbox/mapbox-gl-js/pull/2922 right now.

File: dist/mapbox-gl.css [hunk 1]
```diff
@@ -21,13 +21,13 @@
 .mapboxgl-ctrl-top-left,
 .mapboxgl-ctrl-top-right,
 .mapboxgl-ctrl-bottom-left,
-.mapboxgl-ctrl-bottom-right  { position:absolute; }
+.mapboxgl-ctrl-bottom-right  { position:absolute; pointer-events:none; z-index:2; }
```
Comment (lucaswoj) [inline dist/mapbox-gl.css]: What prompted the `z-index` change?

Comment (yunluyl) [inline dist/mapbox-gl.css]: I saw in leaflet they have z-index defined for the controllers.

.leaflet-control {
	position: relative;
	z-index: 800;
	pointer-events: visiblePainted; /* IE 9-10 doesn't have auto */
	pointer-events: auto;
	}
.leaflet-top,
.leaflet-bottom {
	position: absolute;
	z-index: 1000;
	pointer-events: none;
	}

Then I saw the popup tip has z-index 1. z-index of 2 will make sure the controllers are on top of the popup tip.

.mapboxgl-popup-tip {
    width: 0;
    height: 0;
    border: 10px solid transparent;
    z-index: 1;
}

**Comment by lucaswoj** [comment]
Looks good! Thank you for the contribution @yunluyl!

**Pull request merged**